docs/jwst/introduction.rst

```diff
@@ -412,60 +433,39 @@ logging level designations of ``DEBUG``, ``INFO``, ``WARNING``, ``ERROR``, and
 ``CRITICAL``. Only messages at or above the specified level
 will be displayed.
 
+.. _`Configuration Files`:
 
 Configuration Files
 ===================
 
-Configuration (.cfg) files can be used to specify parameter values
-when running a pipeline or individual steps, as well as for
-specifying logging options.
-
-You can use the ``collect_pipeline_cfgs`` task to get copies of all the cfg
-files currently in use by the jwst pipeline software. The task takes a single
-argument, which is the name of the directory to which you want the cfg files
-copied. Use '.' to specify the current working directory, e.g.
-::
+Configuration files can be used to specify parameter values when running a
+pipeline or individual steps. For JWST, configuration files are retrieved from
+CRDS, just as with other reference files. If there is no match between a step,
+the input data, and CRDS, the coded defaults are used. These values can be
+overridden either by the command line options, as previously described, and by a
+local configuration file.
 
- $ collect_pipeline_cfgs .
+A configuration file should be used when there are parameters a user wishes to
+change from the default/CRDS version, but will not be changed on every run of
+the step. To create a configuration file with the default
+values, use the command: ::
 
-Each step and pipeline has their own cfg file, which are used to specify
-relevant parameter values. For each step in a pipeline, the pipeline cfg file
-specifies either the step's arguments or the cfg file containing the step's
-arguments.
+ strun <step.class> --save-parameters <filename.asdf>
 
-The name of a file in which to save log information, as well as the desired
-level of logging messages, can be specified in an optional configuration file
-"stpipe-log.cfg". This file must be in the same directory in which you run the
-pipeline in order for it to be used. If this file does not exist, the default
-logging mechanism is STDOUT, with a level of INFO. An example of the contents
-of the stpipe-log.cfg file is:
+For example, to get the parameters for the jump step, use:
 ::
 
-    [*]
-    handler = file:pipeline.log
-    level = INFO
+ strun jwst.jump.JumpStep --save-parameters jump_pars.asdf
 
-which specifies that all log messages will be directed to a file called
-"pipeline.log" and messages at a severity level of INFO and above will be
-recorded.
-
-For a given step, the step's cfg file specifies parameters and their default
-values; it includes parameters that are typically not changed between runs.
-Parameters that are usually reset for each run are not included in the cfg file,
-but instead specified on the command line. An example of a cfg file for the
-jump detection step is:
-::
+Once retrieved, the file can be edited, removing parameters that should be left
+at their default/CRDS values, and setting the remaining parameters to the
+desired values. 
```

Comment:
I assume that removing the parameters that the user does not want to change is to make sure those are updated with new parameters with updated default values from CRDS? If so I'd just add that to the text.